os-3.10-multus/scripts/node01.sh

```diff
@@ -84,9 +84,12 @@ while [[ $retry_counter -lt 20  && $os_rc -ne 0 ]]; do
 done
 set -e
 
-/usr/bin/oc create -f /tmp/local-volume.yaml
+# Remove the multus pod to recreate it.
```

Comment:
It says node01 in the file name. Does it mean this will be executed on master node?

Reply:
Yes this will execute only on the master node and remove all the pod related, this way nothing is stuck in containerCreation